controllers/workloads/replicatedstatemachine_controller_test.go

```diff
@@ -20,6 +20,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 package workloads
 
 import (
+	"github.com/apecloud/kubeblocks/pkg/controller/rsm2"
+	viper "github.com/apecloud/kubeblocks/pkg/viperx"
```

Comment:
goimport format

Reply:
fixed in: [851aed9](https://github.com/apecloud/kubeblocks/pull/6776/commits/851aed915d2d9441ac101643776390a6417f5ad1)